src/wurf/git.py

```diff
@@ -22,7 +23,7 @@ def cmd_and_log(self, *args, **kwargs):
         :param kwargs: Keyword arguments passed to ctx.cmd_and_log(...)
         """
         args = [self.git_binary] + list(args)
-        return cmd_and_log(args, **kwargs)
+        return self.ctx.cmd_and_log(args, **kwargs)
```

Comment:
I think what @petya2164 means it that we should remove this function all together - I will do that